ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -121,7 +121,9 @@ Launch ${dashboard_app} From RHODS Dashboard Link
       Click Link   xpath:${CARDS_XP}//*[text()='${splits[0]} ']/../..//a
   ELSE
       IF    "${dashboard_app}" == "Jupyter"
-          Click Link    xpath://div[contains(@class,'pf-v5-l-gallery')]/div[contains(@class,'pf-v5-c-card')]/div[@class="pf-v5-c-card__title"]//span[text()="${dashboard_app}"]/../../..//div[contains(@class,"pf-v5-c-card__footer")]/a
+          # Click Link    xpath://div[contains(@class,'pf-v5-l-gallery')]/div[contains(@class,'pf-v5-c-card')]/div[@class="pf-v5-c-card__title"]//span[text()="${dashboard_app}"]/../../..//div[contains(@class,"pf-v5-c-card__footer")]/a
+          # Workaround for broken 2.6 build
+          Click Link    xpath=//div[@id="jupyter"]//a[@class="odh-card__footer__link"]
```

Comment:
Is this workaround still needed with the latest build? :thinking:

Reply:
It would be needed with the "latest" build I have. Maybe there's something more latest?

Reply:
Does that mean that we should leave a comment in the code?

Reply:
If you do a workaround that's supposed to be temporary, it's always better to keep some note in the code.

Reply:
We can wait to merge until we get an official RC with this fixed and I can remove it from the code, but this was needed to get the test running at all with the current RC

Reply:
@lugi0 could you please check this is still required with the latest RC?

Reply:
It is not and it has been removed